scout/adapter/mongo/panel.py

```diff
@@ -48,8 +43,15 @@ def load_panel(self, parsed_panel):
 
         self.add_gene_panel(panel_obj)
 
-    def load_omim_panel(self, api_key, institute=None):
-        """Create and load the OMIM-AUTO panel"""
+    def load_omim_panel(self, genemap2_lines, mim2gene_lines, institute=None):
+        """Create and load the OMIM-AUTO panel
+
+        If the panel already exists, update with new information and increase version
+
+        Args:
+            genemap_lines(iterable(str)): The genemap2 file information
+            mim2gene_lines(iterable(str)): The mim2genes file information
```

Comment:
Here you could add a comment for institute param

Reply:
Fixed!